uspace/lib/c/generic/rtld/module.c

```diff
@@ -293,14 +299,19 @@ void module_load_deps(module_t *m, mlflags_t flags)
 			dm = module_find(m->rtld, dep_name);
 			if (!dm) {
 				dm = module_load(m->rtld, dep_name, flags);
-				module_load_deps(dm, flags);
+				errno_t rc = module_load_deps(dm, flags);
+				if (rc != EOK) {
+					return EOK;
```

Comment:
This looks suspicious, should it be return rc?

Reply:
yes, you're right